src/formatDate.js

```diff
@@ -51,6 +51,35 @@
 
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+  const newDate = [];
+  const oldSeparator = fromFormat[3];
+  const newSeparator = toFormat[3];
+
+  const arrDate = date.split(oldSeparator);
+
+  const objDate = {};
+
+  for (let i = 0; i <= 2; i++) {
```

Comment:
```suggestion
  for (let i = 0; i < 3; i++) {
```

Try to use same logic across your code. Or you loop through array keys before some index, or inclusive (in identical use cases).